utils/src/main/java/org/robolectric/util/Scheduler.java

```diff
@@ -345,6 +345,7 @@ public synchronized Duration getLastScheduledTaskTime() {
    *     Use {@link #setIdleState(IdleState)} instead to explicitly set the state.
    */
   @Deprecated
+  @SuppressWarnings("InlineMeSuggester")
   public void idleConstantly(boolean shouldIdleConstantly) {
```

Comment:
This method is overridden by other classes and If add the `@lnline` annotation here, it throws an exception at compile time. So basically it's kind of giving a false positives. So, I suppressed it